src/Microsoft.DotNet.Build.Tasks.Feed/IFeed.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    interface IFeed<T>
```

Comment:
I'm not sure I fully understand the point of these interfaces. Who do you expect to use them? What other implementations do you expect?

Reply:
My thinking was we might use different backing sources eg. MyGet, VSTS, Azure. I can get rid of this if needed.

Reply:
It feels like a little bit of over-engineering at this point.